device-types/Cisco/N540X-12Z16G-SYS-D.yaml

```diff
@@ -0,0 +1,77 @@
+---
+manufacturer: Cisco
+model: N540X-12Z16G-SYS-D
+slug: cisco-n540x-12z20g-sys-d
+part_number: N540X-12Z16G-SYS-D
+u_height: 1
+is_full_depth: false
+weight: 5.9
+weight_unit: kg
+module-bays:
+  - name: PM0
+    position: 0/PM0
+    description: Power module
```

Comment:
Hi Thank you for your contribution,

This looks like it has a fixed power supply if so this should be added under `power-ports`